### PR TITLE
Add instructions for libcrypto in ubuntu getting started docs.

### PR DESCRIPTION
**Pull request by SajayAntony**
@danroth27 Addressing aspnet/home#846

### CODE REVIEW

File: aspnet/getting-started/installing-on-linux.rst [hunk 1]
```diff
@@ -98,6 +98,11 @@ The NuGet.Config file should look something like the following
 
 You should now be able to restore packages from both the official public feed on https://nuget.org and also from the ASP.NET 5 dev builds.
 
+Nuget restore also checks the hash on the package. So ensure that you have `libcrypto` available. To do this run:
```
Comment (danroth27) [inline aspnet/getting-started/installing-on-linux.rst]: The text says to install libcrypto but the command installs libssl?

**Comment by danroth27** [comment]
@glennc @shirhatti

**Comment by SajayAntony** [comment]
@danroth27 I can fix it to just libssl but i dont know if that would be enough. Or do you want me to just rename it to install libssl-dev?

File: aspnet/getting-started/installing-on-linux.rst (hunk 1 above)
Comment (anurse) [inline aspnet/getting-started/installing-on-linux.rst]: We also shouldn't need the `-dev` package, since we just need the libraries.

Comment (SajayAntony) [inline aspnet/getting-started/installing-on-linux.rst]: Ok will update to just libssl. 
Thanks

**Comment by anurse** [comment]
Let's just make it `libssl`. Though we need to verify that it works. I highly doubt anything is using the header files provided by `libssl-dev` but it would be good to verify.

**Pull request closed without merging**